tox.ini

```diff
@@ -87,6 +87,7 @@ commands = python -m pip list --format=columns
 [flake8]
 max-complexity = 22
 max-line-length = 99
+ignore = E203, W503
```

Comment:
is PEP8 starting to disagree with black?

Reply:
No, pycodestyle disagrees with PEP 8.

Reply:
I tend to believe you here :)